src/dictionary/mod.rs

```diff
@@ -307,7 +300,7 @@ mod tests {
 
         let mut serialize = Module::new(mock);
         let buffer = serialize.serialize(TEXT);
-        if let Err(_) = serialize.save_schema_to_file(PATH) {
+        if let Err(_) = serialize.save_schema_to_file(path) {
```

Comment:
Tests are run parallelly and this test was  writing to file that the previous test was reading from.